remoteworkitem/scheduler.go

```diff
@@ -11,11 +12,13 @@ import (
 
 // TrackerSchedule capture all configuration
 type trackerSchedule struct {
-	TrackerID   int
-	URL         string
-	TrackerType string
-	Query       string
-	Schedule    string
+	TrackerID      int
+	URL            string
+	TrackerType    string
+	TrackerQueryID int
+	Query          string
+	Schedule       string
+	LastUpdated    *time.Time
```

Comment:
It is really the `updated` time of the last fetched item. So, current name is fine, I think